Jenkinsfile

```diff
@@ -91,22 +91,22 @@ try {
 
             cmake = 'cmake -DCI_BUILD=ON'
 
-            // We don't use unity builds with clang-tidy (see below) and GCC 11 builds as it triggers a GoogleTest
+            // We don't use unity builds with clang-tidy (see below) and GCC 13 builds as it triggers a GoogleTest
```

Comment:
The second part is obsolete. We only do not use unity builds with clang-tidy.

Reply:
Thank you!